flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/RowDataToJsonConverters.java

```diff
@@ -272,8 +271,10 @@ private RowDataToJsonConverter createMapConverter(
 						case DROP:
 							continue;
 						case FAIL:
-							throw new RuntimeException("Map key is null, please have a check."
-								+ " You can setup null key handling mode to drop entry or replace with a no-null literal.");
+							throw new RuntimeException(String.format(
+								"JSON format doesn't support to serialize map data with null keys. "
+									+ "You can drop null key entries or encode null in literals by specifying %s option.",
+								JsonOptions.MAP_NULL_KEY_LITERAL.key()));
```

Comment:
Sorry, it should be `MAP_NULL_KEY_MODE` option.

Reply:
done~